applications/StructuralMechanicsApplication/custom_elements/timoshenko_beam_element_2D2N.h

```diff
@@ -0,0 +1,534 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:  Alejandro Cornejo
+//
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "custom_utilities/structural_mechanics_element_utilities.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class TimoshenkoBeamElement2D2N
+ * @ingroup StructuralMechanicsApplication
+ * @brief This is the Timoshenko beam element of 2 nodes. Reference: Felippa and Oñate,
+ * "Accurate Timoshenko Beam Elements For Linear Elastostatics and LPB Stability",
+ * Archives of Comp. Methods in Eng. (2021) 28:2021-2080
+ * @author Alejandro Cornejo
+ */
+class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) TimoshenkoBeamElement2D2N
```

Comment:
regarding the naming, to be consistent with the existing ones, how about `LinearTimoshenkoBeamElement2D2N`?